### ISSUE TITLE
Manual Study Order Feature for lab home pages

### ISSUE THREAD

**Issue by ianchandlercampbell** [lookit/lookit-api#1024]
# TL;DR
CRADLE: Create a feature that allows labs to manually order their studies on the lab home page created for #620. 

# Narrative

From Ian: It will be helpful to visually show participants the suggested order for study completion. We will also need this feature to work with #625's eligibility criteria modification, as we want this order to adapt to remaining studies for the particular participant. 

As a researcher, I want to be able to assign priority to individual studies in my lab, rather than accepting the random/hashed ordering that's automatically produced on the Search and Lab pages. 

# Acceptance Criteria

- [x] New field appears on Edit Study page allowing a 'priority' integer between 1 and 99 to be assigned. This corresponds to a new variable on a study object called 'study_queue_priority.' 
- [x] By default, all studies (including existing studies) are given a priority of 1 (highest) - the logic here is that, if a lab is not ranking studies, a nefarious lab member would have to modify someone *elses* study, rather than their own, to try and sneakily jump ahead of the other studies. 
- [x] On Search and Lab pages, studies continue to be displayed according to whether they are discoverable, and if the user selects from the drop downs, according to if a particular child is eligible, if the study takes place on or off Lookit, etc. 
- [x] On the main Search page, sorting continues to be by hash only (status quo). 
- [x] On the Lab page, sorting is first by priority, and then by hash.  This means that if a set of studies-to-be-shown contains three with priority 1 and two with priority 10, then they would be displayed as [priority 1 studies ordered by hash]-[priority 10 studies ordered by hash]

# Implementation Notes

It's likely that accomplishing the final 2 criteria will require modifying existing sorting code, and we should proceed with the idea that future changes to sorting could come up in the future - so for instance this might involve defining a set of rankings across both of these page views, where the Search page is set to ["Sort by hash = first", "Sort by lab priority = "no"], and the Lab page is set to ["Sort by lab priority = first", "Sort by hash = second"]
